v2g-liberty/rootfs/root/appdaemon/apps/v2g-liberty/v2g_liberty.py

```diff
@@ -694,19 +694,19 @@ async def __ask_user_dismiss_event_or_not(self, v2g_event: dict):
             self.log(f"__ask_user_dismiss_event_or_not, unexpected call for event with hash_id '{v2g_event['hash_id']}' "
                      f"as car is already disconnected.")
 
-    async def __cancel_charging_timers(self):
-        # self.log(f"cancel_charging_timers - scheduling_timer_handles length: {len(self.scheduling_timer_handles)}.")
+    def __cancel_charging_timers(self):
+        self.log(f"cancel_charging_timers - scheduling_timer_handles length: {len(self.scheduling_timer_handles)}.")
```

Comment:
Are you planning to keep this logging enabled?

Reply:
Yes, it is useful for debugging.